addons/binding/org.openhab.binding.vitotronic/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,489 @@
+        <label>Party</label>
+        <description>Managed Temperatur</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="false"/>
+    </channel-type>
+    
+     <channel-type id="save_temp">
+        <item-type>Number</item-type>
+        <label>Save</label>
+        <description>Managed Temperatur</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="false"/>
+    </channel-type>
+    
+    <channel-type id="room_temp">
+        <item-type>Number</item-type>
+        <label>Room</label>
+        <description>Managed Temperatur</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="false"/>
+    </channel-type>
+    
+    <channel-type id="gradient">
+        <item-type>Number</item-type>
+        <label>Gradient</label>
+        <description>The gradient relativ to outseite temperature.</description>
+        <state pattern="%.1f" readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="niveau">
+        <item-type>Number</item-type>
+        <label>Niveau</label>
+        <description>The niveau relativ to outseite temperature.</description>
+        <state pattern="%.1f" readOnly="true"/>
+    </channel-type>
+ 
+    
+    <!-- Channel Type for solar -->
+
+     <channel-type id="collector_temp">
+        <item-type>Number</item-type>
+        <label>Collector</label>
+        <description>Temperatur sensor</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="true"/>
+    </channel-type>
+ 
+    <channel-type id="storagetank_temp">
+        <item-type>Number</item-type>
+        <label>Storagetank</label>
+        <description>Temperatur sensor</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="bufferload">
+        <item-type>Switch</item-type>
+        <label>Buffer load pump</label>
+        <description> state</description>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="loadsuppression">
+        <item-type>Switch</item-type>
+        <label>Load suppression</label>
+        <description> state</description>
+        <state readOnly="true"/>
+    </channel-type>
+    
+     
+    <channel-type id="producedheat">
+        <item-type>Number</item-type>
+        <label>Produced heat</label>
+        <description>Produced heat since starting solar system</description>
+        <state pattern="%.2f KWh" readOnly="true"/>
+    </channel-type>
+    
+    <!-- single types -->
+    
+    
+    <channel-type id="temperature">
+        <item-type>Number</item-type>
+        <label>Temperature</label>
+        <description>Temperatur sensor</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="value">

```

Comment:
The idea:
A heating system basicly works with 3 Things: Temperature sensors, pumps and valves.
The channel-types are temperature, pump (Switch, already define) and value (of the valve).
I have made the definition of the main things for easy using the binding. I thing that for the most usage of the binding enough. 
But if the heading systems has more "basic" things so it is posible to add this things on the adapter side (xml-file) without changing the binding. This will improve the flexibility.

I going to describe this in the documentation of the binding.